model/atmosphere/diffusion/src/icon4py/model/atmosphere/diffusion/diffusion.py

```diff
@@ -337,9 +337,9 @@ def __init__(
         self.fac_bdydiff_v: Optional[float] = None
         self.bdy_diff: Optional[float] = None
         self.nudgezone_diff: Optional[float] = None
-        self.edge_params: Optional[h_grid.EdgeParams] = None
-        self.cell_params: Optional[h_grid.CellParams] = None
-        self._horizontal_start_index_w_diffusion: gtx.int32 = 0
+        self.edge_params: Optional[geometry.EdgeParams] = None
+        self.cell_params: Optional[geometry.CellParams] = None
+        self._horizontal_start_index_w_diffusion: gtx.int32 = gtx.int32(0)
```

Comment:
just  a question for me and an excuse for testing PR reviews in VScode:
should we always initialise (integers? why not the -5 float a few lines above?) like this?